src/esp/bindings_js/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+// All of the files will be outputted in build_js
+const buildRootPath = path.resolve(__dirname).replace("src", "build_js");
```

Comment:
Would be nice to avoid the use of build_js here. Maybe using cmake's configure_file?